src/Entity/ShipmentTotalWeight.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+namespace Ups\Entity;
+
+use DOMDocument;
+use DOMElement;
+use Ups\NodeInterface;
+use Ups\Entity\UnitOfMeasurement;
+
+class ShipmentTotalWeight implements NodeInterface
+{
+
+    /**
+    * @var UnitOfMeasurement
+    **/
+    private $unitOfMeasurement;
+
+    /**
+    * @var weight
+    **/
+    private $weight;
+
+    public function __construct($response = null)
+    {
+        if (null !== $response) {
+            if (isset($response->unitOfMeasurement)) {
+                $this->setUnitOfMeasurement(new UnitOfMeasurement($response->unitOfMeasurement));
+            }
+            if (isset($response->Weight)) {
+                $this->setWeight($response->Weight);
+            }
+        }
+    }
+
+    /**
+     * @param null|DOMDocument $document
+     *
+     * @return DOMElement
+     */
+    public function toNode(DOMDocument $document = null)
+    {
+        if (null === $document) {
+            $document = new DOMDocument();
+        }
+
+        $node = $document->createElement('ShipmentTotalWeight');
+        if ($this->getUnitOfMeasurement()) {
+            $node->appendChild($this->getUnitOfMeasurement()->toNode($document));
+        }
+
+        $node->appendChild($document->createElement('Weight', $this->getWeight()));
+        
+        return $node;
+    }
+
+    /**
+     * @return UnitOfMeasurement
+     */
+    public function getUnitOfMeasurement()
+    {
+        return $this->unitOfMeasurement;
+    }
+
+    /**
+     * @param UnitOfMeasurement $unitOfMeasurement
+     */
+    public function setUnitOfMeasurement(UnitOfMeasurement $unitOfMeasurement)
+    {
+        $this->unitOfMeasurement = $unitOfMeasurement;
+    }
+
+    /**
+     * @return int
+     */
+    public function getWeight()
+    {
+        return $this->weight;
+    }
+    
+    /**
+     * @param number $weight
```

Comment:
I hope I'm replying to the right comment here. I'm sorry but I can't move this to it's own line or else you won't be able to get rates. If we can guarantee both requests will be merged at the same then I can if not then it will be an issue.

 I can attempt to rename the pull request if that would help.